modules/microshift-install-rhel-image-prepare-kickstart.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-running-bootc-image-in-VM.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-install-rhel-image-prepare-kickstart_{context}"]
+= Preparing the kickstart file
+
+.Prerequisites
+
+* You are logged in to the physical hypervisor host using the credentials.
+* You need to have Sudo permissions configured.
+
+.Procedure
+
+. Set variables pointing to secret files that are included in kickstart.ks for gaining access to private container registries:
++
+[source,terminal]
+----
+AUTH_CONFIG=~/.quay-auth.json
+PULL_SECRET=~/.pull-secret.json
+IMAGE_REF="quay.io/<myorg>/<mypath>/microshift-4.17-bootc" <1>
+----
+<1> Replace _myorg/mypath_ with your remote registry organization name and path.
+. Run the following commands to create the kickstart.ks file to be used during the virtual machine installation:
++
+[source,terminal]
+----
+Rcat > kickstart.ks <<EOFKS
+lang en_US.UTF-8
+keyboard us
+timezone UTC
+text
+reboot
+
+# Partition the disk with hardware-specific boot and swap partitions, adding an
```

Comment:
```suggestion
. Partition the disk with hardware-specific boot and swap partitions, adding an LVM volume that contains a 10GB+ system root. 
+
[source,terminal]
<command here}
```
etc for each step

Reply:
Please, resolve this comment. It cannot be implemented as suggested.